[Bug] Unable to build the project, build instructions are likely outdated

**Describe the bug**

I followed the build instructions from `README.md` file, but was not able to execute `mage build` command successfully.

```
[user@tbd] ➜  ~/tbd/ssi-service git:(main) mage build                                                                                                                                      INSERT
Building project...
# github.com/TBD54566975/ssi-sdk/oidc/issuance
../../go/pkg/mod/github.com/!t!b!d54566975/ssi-sdk@v0.0.3-alpha.0.20230420000447-6d87cd195314/oidc/issuance/metadata.go:17:37: undefined: strings.CutPrefix
Error: running "go build -tags jwx_es256k ./..." failed with exit code 2
```

**To Reproduce**

Follow the [build instructions](https://github.com/TBD54566975/ssi-service#build--test)

**Expected behavior**

Successful compilation.

**Supporting Material**

None

**Environment (please complete the following information):**
 - OS: Linux Fedora 37
 - Go: 1.19.8

**Additional context**

I think that the problem happens because [CutPrefix function was introduced in Go version 1.20](https://golang.google.cn/pkg/strings/#CutPrefix), whereas I have 1.19.8 installed, which is the latest version I can get for my OS with `sudo dnf -y install go` command.

Does `ssi-service` require Go 1.20, which is only 3 months old? If yes, then [Developer prerequisites](https://github.com/TBD54566975/ssi-service/blob/main/CONTRIBUTING.md#development-prerequisites) are out of date, because they indicate Go 1.17 as a requirement.  The [README](https://github.com/TBD54566975/ssi-service/blob/main/README.md?plain=1#L2) shows Go version 1.19.4.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -14,8 +14,8 @@ This guide is for you.
 
 | Requirement | Tested Version | Installation Instructions                             |
 |-------------|----------------|-------------------------------------------------------|
-| Go          | 1.17.6         | [go.dev](https://go.dev/doc/tutorial/compile-install) |
-| Mage        | 1.12.1         | [magefile.org](https://magefile.org/)                 |
+| Go          | 1.20.3         | [go.dev](https://go.dev/doc/tutorial/compile-install) |
+| Mage        | 1.13.0         | [magefile.org](https://magefile.org/)                 |
 
 ### Go
 
@@ -25,7 +25,7 @@ You may verify your `go` installation via the terminal:
 
 ```
 $> go version
-go version go1.17.6 darwin/amd64
+go version go1.20.3 darwin/amd64
 ```
 
 If you do not have go, we recommend installing it by following the instructions on the [Go website](https://go.dev/doc/install).
@@ -46,9 +46,9 @@ You may verify your `mage` installation via the terminal:
 
 ```
 $> mage --version
-Mage Build Tool 1.12.1
-Build Date: 2021-12-15T21:00:02Z
-Commit: 2f1ec40
+Mage Build Tool v1.13.0-6-g051a55c
+Build Date: 2022-05-02T19:53:34-07:00
+Commit: 051a55c
 built with: go1.17.6
 ```
 

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
 [![godoc ssi-service](https://img.shields.io/badge/godoc-ssi--service-blue)](https://github.com/TBD54566975/ssi-service)
-[![go version 1.19.4](https://img.shields.io/badge/go_version-1.19.4-brightgreen)](https://go.dev/)
+[![go version 1.20.3](https://img.shields.io/badge/go_version-1.20.3-brightgreen)](https://go.dev/)
 [![license Apache 2](https://img.shields.io/badge/license-Apache%202-black)](https://github.com/TBD54566975/ssi-service/blob/main/LICENSE)
 [![issues](https://img.shields.io/github/issues/TBD54566975/ssi-service)](https://github.com/TBD54566975/ssi-service/issues)
 ![push](https://github.com/TBD54566975/ssi-service/workflows/ssi-service-ci/badge.svg?branch=main&event=push)

